Content/default/README.md

```diff
@@ -17,6 +18,15 @@ Before you run the project **for the first time only** you must install dotnet "
 dotnet tool restore
 ```
 
+To check for updates, use
```

Comment:
I'm not sure that this additional section is required - it's just standard `dotnet tool` information.

Reply:
correct, is just what you should know anyways. I felt it was kind of helpful for me lately. But you are right.

Reply:
removed this section